src/support/command/handle-response.js

```diff
@@ -39,10 +39,10 @@ const parseResponseError = ({ content }) => hasJsonStructure(content)
   : errorMsg.unknown()
 
 const handleErrors = error => {
-  const cliError = isError(error)
-    ? error
-    : parseResponseError(error)
-
+  if (isError(error)) {
```

Comment:
Again we were logging the stacktrace of the error here so. Better solution is probably 
```
const cliError = isError(error) ? error.message : parseResponseError(error)
```
now that I think about it.